cosmic-usage-api/src/test/java/com/github/missioncriticalcloud/cosmic/api/usage/repositories/jdbc/PublicIpsJdbcRepositoryIT.java

```diff
@@ -0,0 +1,33 @@
+package com.github.missioncriticalcloud.cosmic.api.usage.repositories.jdbc;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import com.github.missioncriticalcloud.cosmic.usage.core.model.PublicIp;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.test.context.ActiveProfiles;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest
+@ActiveProfiles("local")
+public class PublicIpsJdbcRepositoryIT {
+
+    @Autowired
+    private PublicIpsJdbcRepository publicIpsRepository;
```

Comment:
We shouldn't lock the test to any specific implementation of a repository, JDBC in this case. I would suggest instead to point to the repository's interface (`PublicIpsRepository`), rename the test class to `PublicIpsRepositoryIT` and move the class one package up.